cephfs/cephfs_test.go

```diff
@@ -270,6 +270,45 @@ func TestGetSetConfigOption(t *testing.T) {
 	assert.Equal(t, origVal, currVal)
 }
 
+func TestParseConfigArgv(t *testing.T) {
+	mount, err := CreateMount()
+	require.NoError(t, err)
+	require.NotNil(t, mount)
+	defer func() { assert.NoError(t, mount.Release()) }()
+
+	origVal, err := mount.GetConfigOption("log_file")
+	assert.NoError(t, err)
+
+	err = mount.ParseConfigArgv(
```

Comment:
what happens on an empty `[]string{}`?

Reply:
It would have panicked. There's now a check in place.